src/data/consumer-price-index/api/article/index.ts

```diff
@@ -0,0 +1,20 @@
+import { getTopTwoArticle } from '../../utils';
+import { NextApiHandler } from 'next';
+
+const getArticle: NextApiHandler = async (_, res) => {
+  const articles = await getTopTwoArticle();
+
+  if (Object.values(articles).some(v => v[0] == null)) {
+    return res.status(404).end();
+  }
+
+  return res.status(200).json(articles);
+};
+
+const handler: NextApiHandler = async (req, res) => {
+  if (req.method === 'GET') {
+    return getArticle(req, res);
```

Comment:
```suggestion
    return getArticles(req, res);
```
마이너이긴 한데...단수/복수 여부로 인해 Response DTO가 달라지는 경우가 많아서 가급적 명확하게 구분해주는게 좋긴합니다. `getTopTwoArticle` 도 마찬가지에유